hudi-spark/src/main/scala/org/apache/hudi/IncrementalRelation.scala

```diff
@@ -92,17 +92,8 @@ class IncrementalRelation(val sqlContext: SQLContext,
     StructType(skeletonSchema.fields ++ dataSchema.fields)
   }
 
-
-  private val filters = {
-    if (optParams.contains(DataSourceReadOptions.PUSH_DOWN_INCR_FILTERS_OPT_KEY)) {
```

Comment:
Why we remove the if/else block?

Reply:
> Why we remove the if/else block?

because if the map does contain `DataSourceReadOptions.PUSH_DOWN_INCR_FILTERS_OPT_KEY`, we will set it to the default value, and the default value will result in an empty array, same as the else block. so there is no need to check it.